code/modules/mob/living/silicon/robot/drone/drone_abilities.dm

```diff
@@ -47,4 +47,55 @@
 	if(M.a_intent == I_HELP)
 		get_scooped(M)
 
-	..()
\ No newline at end of file
+	..()
+
+/mob/living/silicon/robot/drone/verb/customize()
+	set name = "Customize Chassis"
+	set desc = "Reconfigure your chassis into a customized version."
+	set category = "Drone"
+
+	if(!custom_sprite) //Check to see if custom sprite time, checking the appopriate file to change a var
+		var/file = file2text("config/custom_sprites.txt")
+		var/lines = splittext(file, "\n")
+
+		for(var/line in lines)
+		// split & clean up
+			var/list/Entry = splittext(line, ":")
+			for(var/i = 1 to Entry.len)
+				Entry[i] = trim(Entry[i])
+
+			if(Entry.len < 2 || Entry[1] != "drone")
+				continue
+
+			if (Entry[2] == ckey) //Custom holograms
+				custom_sprite = 1  // option is given in hologram menu
+
+	if(!custom_sprite)
+		to_chat(src, "<span class='warning'>Error 404: Custom chassis not found. Revoking customization option.</span>")
+	else
+		icon = 'icons/mob/custom_synthetic/custom-synthetic.dmi'
+		icon_state = "[ckey]-drone"
+		to_chat(src, "<span class='notice'>You reconfigure your chassis and improve the station through your new aesthetics.</span>")
+	verbs -= /mob/living/silicon/robot/drone/verb/customize
```

Comment:
If you wanted things smoother you could put this in the drone's `login` but it's not really important